server/channels/jobs/mobile_versions/scheduler.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package mobile_versions
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/channels/jobs"
+)
+
+const schedFreq = 24 * time.Hour
+
+func MakeScheduler(jobServer *jobs.JobServer) *jobs.PeriodicScheduler {
+	isEnabled := func(cfg *model.Config) bool {
+		return *cfg.MetricsSettings.EnableClientMetrics
+	}
+	return jobs.NewPeriodicScheduler(jobServer, model.JobTypeActiveUsers, schedFreq, isEnabled)
```

Comment:
Is it not possible to reuse the current daily job that sends daily server telemetry to Rudder so we don't have to add a new one?

https://github.com/mattermost/mattermost/blob/6cece9faa6eae10b97e319e8cdeab5477708a570/server/platform/services/telemetry/telemetry.go#L1389

Reply:
The telemetry job is encapsulated into an interface that doesn't have (apparently) access to the metrics interface.